src/main/java/com/google/codeu/data/User.java

```diff
@@ -1,13 +1,24 @@
 package com.google.codeu.data;
 
+//Provides the capability to save the data the user submits, 
```

Comment:
Given that this is a comment on the class, you should use javadoc style comments, eg:

```java
/**
 * Represents the data associated with a given user.
 */
```

Note that it starts with /**, which has a special meaning to Java compilers.